civil.md

```diff
@@ -34,24 +36,36 @@ Before we talk about how to name them, we should be clear on what things we are
+
 **Con:** Gregorian requires knowledge of the fact that the *proleptic gregorian calendar* is what most of the western world uses. This is not knowledge we can assume.
 
 **Con:** `CivilTime` does not in any way depend on the gregorian calendar. In fact it is a secondary characteristic: *the midnight to midnight* day definition. Even in cultures that use the gregorian calendar, this definition may not apply (think Ethiopia). So gregorian would be factually wrong for times and misleading in combination with some calendars.
 
 ## Local
 
-**Pro:** Local is the prefix that many programming languages use for this type of object. It is therefore very intuitive to many users.
+**Pro:** Local is the prefix that many programming languages use for exactly these types of object for around 20 years. It is therefore very intuitive to many users and widely defined in internet searches and help forums.
+
+**Pro:** ISO-8601 used "local time" over 20 years ago for the concept:
+
+> 2.1.16 local time: locally applicable time of day such as standard time of day, or a non-UTC based time of day"
 
-**Con:** Local is actually wrong in that it names the one characteristic that these objects do **not** possess: locality.
+The key is that it is "locally applicable", not that it implies a location.
 
-**Con:** Local is the prefix that many languages use. However the `Civil` objects are part of a larger whole with a very specific mental model. Local violates that mental model and by suggesting analogies to other languages it exacerbates that misunderstanding of the meaning and value of these types.
+**Pro:** The term "local time" is in widespread use, both in scholarly articles and common parlance. For example, airline pilots will typically say "when we land the local time will be 7pm".
+
+**Con:** Local names a characteristic that these objects do not possess: locality. But surely this misunderstands how "local" is used in English - as with the airline case, the phrase "local time" doesn't refer to knowledge of the locality, but the absence of needing to worry about it on your watch/phone once it is set correctly. As a related example, talking about your "local shop" doesn't require identifying a specific shop in a specific locality. Instead it is a phrase used to discuss whatever the shop nearest to you is, something that varies depending on where you are.
 
 ## Civil
 
 **Pro:** Civil is an usused prefix that suggests that these objects are used to represent things used by civil society.
 
 **Con:** Because Civil is a hitherto unused prefix, it does not lend itself to easy comprehension without the entire mental model.
 
+**Con:** Some will consider "civil" to be naming the calendar system, effectively bringing all the cons listed in that section above.
+
+**Con:** Relatively few non-English speakers will know what Civil means.
```

Comment:
Why is this a con?  The fact that non-native English speakers are less likely to recognize this word means that they may read the documentation more closely and learn the concept correctly, rather than relying on false presumptions (such as assuming local implies locality).